### PR TITLE
donation validator

### PR DESCRIPTION
**Pull request by KaiNissen**

### CODE REVIEW

File: src/Validation/CanValidateValueObject.php [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace WMDE\Fundraising\Frontend\Validation;
+
+trait CanValidateValueObject {
```
Comment (JeroenDeDauw) [inline src/Validation/CanValidateValueObject.php]: What's the reasoning behind introducing this trait?

**Comment by JeroenDeDauw** [comment]
Follow up: #137

File: src/Validation/CanValidateValueObject.php (hunk 1 above)
Comment (KaiNissen) [inline src/Validation/CanValidateValueObject.php]: Uh, might as well just be a private method.

**Comment by JeroenDeDauw** [comment]
+2 when follow up is merged

**Pull request merged**